omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -321,4 +349,43 @@ private Boolean isNewUser(User user) {
 		return user == null ? true : user.getUserId() == null;
 	}
 	
+	/**
+	 * @return true if email is valid or false otherwise
+	 * @param email
+	 */
+	private boolean isEmailValid(String email) {
+		return StringUtils.hasLength(email) && EmailValidator.getInstance().isValid(email);
+	}
+	
+	/**
+	 * @return an email field
+	 * @param user
+	 */
+	private Field getEmailField() {
```

Comment:
Shouldn't this method return the email field value instead of the field object?

Reply:
i would find it hard to reuse this method it in `handleSubmission` if it was returning the value

Reply:
Oh i see!